pyibisami/run_tests.py

```diff
@@ -128,8 +129,10 @@ def run_tests(**kwargs):
     """Provide a thin wrapper around the click interface so that we can test the operation."""
 
     # Fetch options and cast into local independent variables.
-    test_dir = Path(kwargs["test_dir"])
+    test_dir = Path(kwargs["test_dir"]).resolve()
```

Comment:
Later you get the absolute path of test_dir.  Resolve provides the same functionality but for pathlib.  The choice to go to path lib btw, was that it correctly handles all the operating systems (instead of depended on us to manually string form) and allows us to `glob` directories instead of the nest list comprehensions that you had to get files that ended with a certain extension.  
```
test_dir = op.abspath(test_dir)
```